senator/__main__.py

```diff
@@ -1,8 +1,49 @@
 from senator.utility import *
 from senator.game import *
+import argparse
+import numpy as np
 
-votes, initial_utility = parse_data(load_data(FILE_PATH))
 
-game = Game(initial_utility,votes)
+def main() -> tuple[np.ndarray, np.ndarray, np.ndarray]:
 
-game.run_greedy_search()
\ No newline at end of file
+    parser = argparse.ArgumentParser(
+        description="Run the senator game simulation with specified parameters."
+    )
+
+    parser.add_argument(
+        "--owner_loss",
+        type=float,
+        default=0.8,
+        help="Loss for the owner (default: 0.5)",
+    )
+    parser.add_argument(
+        "--other_loss", type=float, default=0.0, help="Loss for others (default: 0.0)"
+    )
+    parser.add_argument(
+        "--owner_trashold",
+        type=float,
+        default=0.35,
+        help="Threshold for the owner (default: 0.35). Set to -1 to change the model.",
+    )
+
+    args = parser.parse_args()
+
+    votes, initial_utility, is_owner = parse_data(load_data(FILE_PATH))
```

Comment:
ještě jedna věc mě napadla že by mohla být fajn. místo `FILE_PATH` přidat argument `file_path`, a `FILE_PATH` mu nastavit jako default.

Reply:
Já myslel jako argument do toho `parser`u ať to jde nastavit z command line :D

Reply:
ok, upravím